pkg/diff/testdata/smd-deploy-live.yaml

```diff
@@ -120,7 +125,10 @@ spec:
           ports:
             - containerPort: 80
               protocol: TCP
```

Comment:
In the case of both `ports` and `containers`, bot the live and the config yaml contain only one list item. I think we should probably do a test involving multiple list items, ideally with differing orders, to demonstrate that the key matching works correctly.

Reply:
Okay